assets/css/sensei-learning-mode/base.scss

```diff
@@ -2,17 +2,17 @@
 $breakpoint: 783px;
 
```

Comment:
> assets/css/**sensei-course-theme** → assets/css/**sensei-learning-mode**

I think `sensei-learning-mode` can be just `learning-mode` for this directory. 

Just as some background, the original logic for some of this was that the course has a theme ('course theme') and that theme (the _value_ of the field) is either 'default' (site theme) or 'sensei-course-theme' (the theme that we now call learning mode).